src/future/client.rs

```diff
@@ -90,10 +90,10 @@ enum Reactor {
 
 impl fmt::Debug for Reactor {
     fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
-        const HANDLE: &'static &'static str = &"Reactor::Handle";
-        const HANDLE_INNER: &'static &'static str = &"Handle { .. }";
-        const REMOTE: &'static &'static str = &"Reactor::Remote";
-        const REMOTE_INNER: &'static &'static str = &"Remote { .. }";
+        const HANDLE: &'static &str = &"Reactor::Handle";
```

Comment:
Can this outer 'static also be elided? Likewise below.

Reply:
Right. Not sure why I left it... Pushed another commit that fixes it.
However, why not derive `Debug` instead of implementing it manually?

Reply:
Not all of the fields of Reactor had impl'd `Debug` when this was written. If that's no longer the case, then this should absolutely be deriving `Debug`. Feel free to submit another PR with that change. In the meantime, I'll merge this in!

Reply:
Cool, thanks!